run_testlist.py

```diff
@@ -32,972 +33,606 @@
-                    'testvariants': [
-                        (4, 16, 5),
-                        (8, 16, 5),
-                        (16, 16, 5),
-                        (32, 16, 5),
-                        (64, 16, 5),
-                        (128, 16, 5),
-                        (256, 16, 5)
-                    ],
-                    'ppc': 32,
-                    'env_vars': {
-                        'chunk_size': '1048576',
-                        'pool_size': '85G',
-                        'n_file': '200000',
-                        'bytes_read': '3901',
-                        'bytes_write': '3901',
-                        'tree_depth': '0/20',
-                        'oclass': 'RP_3G1',
-                        'dir_oclass': 'RP_3GX',
-                        'sw_time': '60'
-                    },
-                    'enabled': False
-                    }
-                   ]
-
-
-swim_testlist = [{'testcase': 'rebuild_pool_single',
-                  # Number of servers, number of clients, timeout in minutes
-                  'testvariants': [
-                      (2, 1, 10),
-                      (4, 1, 10),
-                      (8, 1, 10),
-                      (16, 1, 10),
-                  ],
-                  'ppc': 32,
-                  'env_vars': {
-                      'pool_size': '85G',
-                      'number_of_pools': '1'
-                  },
-                  'enabled': False
-                  },
-                  {'testcase': 'rebuild_pool_multi',
-                  # Number of servers, number of clients, timeout in minutes
-                  'testvariants': [
-                      (2, 1, 30),
-                      (4, 1, 30),
-                      (8, 1, 15),
-                      (16, 1, 30),
-                  ],
-                  'ppc': 32,
-                  'env_vars': {
-                      'pool_size': '256MiB', # Minimum 16MiB per rank
-                      'number_of_pools': '73'
-                  },
-                  'enabled': False
-                  }
-                 ]
-
-
-swim_ior_testlist = [{'testcase': 'pool_rebuild_50_load',
-                      # Number of servers, number of clients, timeout in minutes
-                      'testvariants': [
-                          (8, 4, 30),
-                      ],
-                      'ppc': 32,
-                      'env_vars': {
-                          'pool_size': '85G',
-                          'number_of_pools': '1',
-                          'chunk_size': '1048576',
-                          'segments': '1',
-                          'xfer_size': '1M',
-                          'block_size': '2656M',
-                          'oclass': 'SX',
-                          'iterations': '1'
-                      },
-                      'enabled': False
-                      }
-                     ]
-
+# TODO refactor to make 'inflight' a variant?
```

Comment:
One idea is to somehow make the params generic enough that mostly any param could be a variant. I have some ideas, but nothing concrete yet